conf/php-fpm.conf

```diff
@@ -1,430 +1,18 @@
-; Start a new pool named 'www'.
```

Comment:
This file follow the [template](https://github.com/YunoHost/example_ynh/blob/master/conf/php-fpm.conf)

I'm not sure it's a good idea to remove all comments for future updates